src/pages/GoverningBody/AddDocumentModal/AddDocumentModal.tsx

```diff
@@ -69,13 +77,17 @@ const AddDocumentModal = (props: Props) => {
         setDisabled(true);
       }
       
+      const isEmptyFile = fileSize !== 0;
+      if (!isEmptyFile)
+      notificationLogic("error", fileIsEmpty());
+
       const isSmaller3mb = fileSize < 3145728;
```

Comment:
I know that this is not your code, but please, remove this magic number and use constant instead